.travis.yml

```diff
@@ -1,20 +1,9 @@
 language: node_js
 sudo: required
-before_install: # if "install" is overridden
```

Comment:
there already is a separate PR addressing this problem :)

Reply:
👍 I thought there might be - I just wanted to be able to check if it would pass the CI tests.